app/assets/stylesheets/show.scss

```diff
@@ -0,0 +1,87 @@
+@import "foundation_and_overrides";
+@import "index";
+$background-color: #088da5;
+$white: #ffffff;
+
+.art-details {
+  background-color: $background-color;
+  box-shadow: 5px 5px 3px black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.